src/main.js

```diff
@@ -215,8 +215,10 @@ $(document).ready(() => {
       handleHoverOpenOption(this.store.get(STORE.HOVEROPEN));
 
       // Immediately closes if click outside the sidebar.
-      $document.on('click', () => {
-        if (!isMouseInSidebar && !isSidebarPinned() && isSidebarVisible()) {
+      $document.on('click keyup', (e) => {
+        const isEscape = e.type === 'keyup' && e.key === 'Escape';
```

Comment:
Did someone request this? Not sure the implication... The more complicated this code is, the more nervous I am.

Reply:
It's just a suggestion.